cf_remote/demo-admin-password.sql

```diff
@@ -0,0 +1,25 @@
+UPDATE
```

Comment:
I believe you'll need to tell the package manager (i.e. `pip`) to install this file somewhere, somehow. Were you able to run it from an installed version of `cf-remote`? Or did you only test it from the source directory?

Yup:
```
$ cf-remote --version master install --demo --bootstrap hub --hub hub
 ---snip---
cf_remote.aramid.ExecutionError: SSH failed on '3.251.85.184' with error 'scp: stat local "/home/larsewi/.venv/lib/python3.12/site-packages/cf_remote/demo-admin-password.sql": No such file or directory
'
```